apps/payments/next/components/HelloWorld/index.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Meta } from '@storybook/react';
+import HelloWorld from '.';
+
+export default {
+  title: 'components/HelloWorld',
+  component: HelloWorld,
+} as Meta;
+
+export const Default = () => {
+  return <HelloWorld />;
+};
+
+// export const Accessible = () => <button>Accessible button</button>;
```

Comment:
Let's remove L13-19 if it's not ready

Reply:
Same as the other comment. Since we are adding a11y to M1 and other PRs have commented out portions as well.